be/src/util/arrow/row_batch.h

```diff
@@ -45,6 +46,9 @@ Status convert_to_arrow_type(const TypeDescriptor& type, std::shared_ptr<arrow::
 Status convert_to_arrow_schema(const RowDescriptor& row_desc,
                                std::shared_ptr<arrow::Schema>* result);
 
+Status get_block_arrow_schema(const vectorized::Block& block,
```

Comment:
It's better to use the same name convert_to_arrow_schema to indicate function overload.